src/DonorProfiles/resources/js/app/components/rest-subscription-table/subscription-row/index.js

```diff
@@ -1,23 +1,23 @@
 const SubscriptionRow = () => {
 	return (
-		<div className="give-donor-profile-table__row">
-			<div className="give-donor-profile-table__column">
-				<div className="give-donor-profile-table__subscription-amount">$200 / Monthly</div>
-				<div className="give-donor-profile-table__subscription-campaign">Save Friar Tuck</div>
+		<div className="give-donor-dashboard-table__row">
+			<div className="give-donor-dashboard-table__column">
+				<div className="give-donor-dashboard-table__subscription-amount">$200 / Monthly</div>
+				<div className="give-donor-dashboard-table__subscription-campaign">Save Friar Tuck</div>
 			</div>
-			<div className="give-donor-profile-table__column">
-				<div className="give-donor-profile-table__subscription-status">
+			<div className="give-donor-dashboard-table__column">
+				<div className="give-donor-dashboard-table__subscription-status">
 					Active
```

Comment:
Can we localize these strings?

Reply:
This component is unused and has been removed, ref: https://github.com/impress-org/givewp/pull/5708/commits/ba86799ecdac13b18d894dffc003e970b25b23d5